src/main/java/org/stellar/sdk/ManageOfferOperation.java

```diff
@@ -19,10 +20,10 @@
   private final Asset selling;
   private final Asset buying;
   private final String amount;
-  private final String price;
+  private final Price price;
   private final long offerId;
 
-  private ManageOfferOperation(Asset selling, Asset buying, String amount, String price, long offerId) {
+  private ManageOfferOperation(Asset selling, Asset buying, String amount, Price price, long offerId) {
```

Comment:
Can we overload the method (leave the existing methods and add a new methods with `Price`) for backward compatibility?